### PR TITLE
Some regularisation of tracer rate reporting.

### PR DESCRIPTION
**Pull request by osae**
This hopefully improves the behavior of tracer concentration summary curves, (F/W)T(IR)C.

### CODE REVIEW

**Comment by alfbr** [comment]
This is highly welcome! The results are now at the point where I believe it makes sense to mark Tracer supported and have it on by default. The only negative impact I can see is that the computational time of Norne goes up a little, but it really is about time we stop cheating there :-)

**Comment by bska** [comment]
jenkins build this please

File: ebos/ecltracermodel.hh [hunk 1]
```diff
@@ -402,11 +407,33 @@ protected:
...
+            // TODO: Some inconsistencies here that perhaps should be clarified. The "offical" rate as reported below is
+            //  occasionally significant different from the sum over connections (as calculated above). Only observed
+            //  for small values, neglible for the rate itself, but matters when used to calculate tracer concentrations.
+            std::size_t well_index = simulator_.problem().wellModel().wellState().wellIndex(well.name());
+            Scalar official_well_rate_total = simulator_.problem().wellModel().wellState().wellRates(well_index)[tr.phaseIdx_];
+
+            rateWellTotal = official_well_rate_total;
+
+            if (rateWellTotal > rateWellNeg) { // Cross flow
+                const Scalar bucketPrDay = 10.0/(1000.*3600.*24.); // ... keeps (some) trouble away
```
Comment (bska) [inline ebos/ecltracermodel.hh:432]: Do we expect to solve tracers in "LAB" environments?  If so, this value might be too large.

**Review by bska — COMMENTED** [review]
The code itself looks okay to me.  I just have a question about the `bucketPerDay` threshold.  It seems a little arbitrary and very much geared towards the reservoir simulation application.  Then, of course, if tracers aren't common elsewhere that may well be a reasonable compromise.

**Comment by alfbr** [comment]
> The code itself looks okay to me. I just have a question about the `bucketPerDay` threshold. It seems a little arbitrary and very much geared towards the reservoir simulation application. Then, of course, if tracers aren't common elsewhere that may well be a reasonable compromise.

I cannot recall tracers being used in a lab setting, so I believe it is safe to assume we have a reservoir setting.

**Comment by bska** [comment]
> I cannot recall tracers being used in a lab setting, so I believe it is safe to assume we have a reservoir setting.

Okay, in that case I'll merge this.

**Pull request merged**